src/components/Kv/KvPillToggle.vue

```diff
@@ -26,45 +26,71 @@ export default {
 	},
 	methods: {
 		pillToggled(key) {
-			this.$emit('pillToggled', key);
+			this.selected = key;
```

Comment:
Couple of notes here:
1. This is actually an anti-pattern in Vue. We only pass data down the component tree using props, not back up. This causes a console error warning us: `Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "selected"`. Basically whenever our parent re-renders any changes made by this line of code will be reset. @emuvente shared a useful link with more info here: http://michaelnthiessen.com/avoid-mutating-prop-directly/
2. As this is a controlled component, we'll update this value in the parent. We can make other changes to make this a mixed (controlled/uncontrolled) component too but for now we can remove this line and read the `pillToggled` event in the parent.

tl;dr: We can delete line 29

Reply:
@ksekhri PR updated per your suggestion.